### ISSUE TITLE
iframes are not focusable in all browsers [mydYmvr]

### ISSUE THREAD

**Issue by WilcoFiers** [auto-wcag/auto-wcag#2319]
## Accessibility support

Some browsers will automatically make any scrollable element focusable to ensure keyboard accessibility. However, the browser does not include these elements in sequential focus navigation when it has a negative number as a tabindex attribute value.

Some browsers restrict scrolling to the content box of elements; while others allow to scroll the full border box, hence including the element’s padding. This results in some elements being scrollable with a browser but not with another.

## ACT Rules

- [Scrollable content can be reached with sequential focus navigation - 0ssw9k](https://www.w3.org/WAI/standards-guidelines/act/rules/0ssw9k/proposed/#accessibility-support)

### Test case 1

Testing SVG role and naming support across different methods

```html
<button>before</button>
<iframe 
	srcdoc="<h1>hello</h1><br><br><br><br><br><br><br><br><br><p>World</p>" 
	height="50"
></iframe>
<button>after</button>
```

#### Test instructions

- Open the example in [codepen](https://codepen.io/wilcofiers/full/mydYmvr)
- Place focus on the "before" button
- Press the `tab` key
- Press and hold the `down` arrow
- Expect the iframe to scroll

### External links

TODO

## Test results

Please report test results in a comment below. Please use the following format:

```md
## Test case 1

- Date: ...
- Operating system: ..., version ...
- Browser: ..., version ...
- Screen reader: ..., version ...
- Test result: Passed / Failed (if failed, how was it different from expected?)
```

**Comment by WilcoFiers** [comment]
## Chrome 135

- Date: 2025-04-08
- Operating system: MacOS, version 14.5
- Test result: Passed

## Firefox 137

- Date: 2025-04-08
- Operating system: MacOS, version 14.5
- Test result: Passed

## Safari 17.5

- Date: 2025-04-08
- Operating system: MacOS, version 14.5
- Test result: Passed

**Comment by WilcoFiers** [comment]
Looks like this works in all major browsers now. We may be able to deprecate this rule.

**Comment by WilcoFiers** [comment]
Sorry, no I'm getting things confused. Iframes aren't failed by this rule, it's scrollable non-frame elements.
